chat/chat.go

```diff
@@ -178,31 +179,44 @@ func main() {
 
 	if *dest == "" {
 		// Set a function as stream handler.
-		// This function  is called when a peer initiate a connection and starts a stream with this peer.
-		// Only applicable on the receiving side.
+		// This function is called when a peer connects, and starts a stream with this protocol.
+		// Only applies on the receiving side.
 		host.SetStreamHandler("/chat/1.0.0", handleStream)
 
-		fmt.Printf("Run './chat -d /ip4/127.0.0.1/tcp/%d/ipfs/%s' on another console.\n You can replace 127.0.0.1 with public IP as well.\n", *sourcePort, host.ID().Pretty())
```

Comment:
Better to replace `ipfs` with `p2p` and use `P_P2P` in `addAddrToPeerstore`

Reply:
Good point!